DTS-STN/Alpha-Site
# Feature/unsubscribe

# Description

[Unsubscribe page](https://trello.com/c/khh5myus)

### Important
This PR introduces two new env variables:
* USER_UNSUBSCRIBE_ENGLISH_TEMPLATE_ID
* USER_UNSUBSCRIBE_FRENCH_TEMPLATE_ID

### /unsubscribe
* Update the content of `/unsubscribe` to match what's in Figma
* Add unit and e2e tests for the page

### api routes and redirects
 * Update `/api/validate` and `/api/unsubscribe` to redirect to reworked versions of `/confirmation`
 * Update `/thankyou` and `/confirmation` to work with both signup and unsubscribe flows

### /lib/utils
* Moved `maskEmail` function to it's own file since I originally was going to use that function across pages, but then I kind of redesigned the pages to reduce duplicate code. Still decided to move this function to make it easier to unit test. I think this function will need more work for error checking though, but that will go in another PR sometime.

## Acceptance Criteria

The unsubscribe functionality works and content matches what's in Figma.

## Test Instructions

1. After completing the signup process (no need to confirm the email though), navigate to `/unsubscribe`
2. Notice the design matches what's in Figma
3. Follow the flow to see that it works as expected


## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
